pkg/omf/util/omf.util_bundle_name.fish

```diff
@@ -0,0 +1,3 @@
+function omf.util_bundle_name -a name_or_url
```

Comment:
I prefer bundle_name or bundlename, as in basename, just because sanitize has unfortunate associations in my mind. Plus, at this point it is somewhat misleading to claim that the function seriously deals with any security concerns.
Anyway, these are my 2 cents. You're the architect, so say the word and it's changed.